Directory.Packages.props

```diff
@@ -69,8 +69,6 @@
     <PackageVersion Include="Microsoft.VisualStudio.Validation"                                      Version="17.8.8" />
     <PackageVersion Include="Microsoft.VisualStudio.XmlEditor"                                       Version="17.13.0-preview-1-35408-014" />
     <PackageVersion Include="Microsoft.VSSDK.BuildTools"                                             Version="18.0.136-preview1" />
-    <PackageVersion Include="System.Threading.Tasks.Dataflow"                                        Version="9.0.0" />
```

Comment:
Why removing the pinned version of Dataflow/other packages

Reply:
They are not needed. .NET 10 SDK reports warnings.